### PR TITLE
Fix cards dnd + Fix blurry icons

### PR DESCRIPTION
**Pull request by vrialland**

### CODE REVIEW

File: static/css/themes/board.css [hunk 1]
```diff
@@ -387,7 +387,9 @@ textarea.expanded {
 
 .card-actions button i[class^="icon-"],
 .card-actions .btn i[class^="icon-"] {
-    margin-right: 3px;
+    margin-right: 5px;
+    position: relative;
+    top: 1px;
```
Comment (rtxm) [inline static/css/themes/board.css]: What does that do?

File: static/css/themes/board.css [hunk 2]
```diff
@@ -841,6 +843,8 @@ div.history th {
 .checklists .checklist .title i.icon-list {
     margin-right: 10px;
     cursor: move;
+    position: relative;
+    top: 1px;
 }
```
Comment (rtxm) [inline static/css/themes/board.css]: What does that do? Does it fix blurry icons or something else?

File: static/css/themes/board.css (hunk 1 above)
Comment (vrialland) [inline static/css/themes/board.css]: As icons are a little bit bigger, they are not properly align with the nearby text, so I had to move them a pixel away.

File: static/css/themes/board.css (hunk 2 above)
Comment (vrialland) [inline static/css/themes/board.css]: Same as above

**Comment by rtxm** [comment]
Ok.
LGTM.

**Pull request merged**